about.html

```diff
@@ -0,0 +1,100 @@
+  <link
+    href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Playfair+Display:wght@400;500;700&family=Poppins:wght@200;400&display=swap"
+    rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css" integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <script defer src="script.js"></script>
+</head>
+<body>
+  <header>
+    <div class="container-fluid deskto-top-nav">
+      <div class="social-media-div">
+        <ul class="social-media">
+          <li class="">
+            <a href="" class="social-media-item"><i class="fa-brands fa-facebook-f"></i></a>
+          </li>
+          <li class=""><a href="#" class="social-media-item"><i class="fa-brands fa-twitter"></i></a>
+          </li>
+          <li class="">
+            <a href="#" class="social-media-item">English</a>
+          </li>
+          <li class="">
+            <a href="#" class="social-media-item">Mypage</a>
+          </li>
+          <li class="">
+            <a href="#" class="social-media-item">Logout</a>
+          </li>
+        </ul>
+      </div>
+    </div>
+    <nav class="navbar">
+      <div class="logo"><a href="./index.html"><img src="img/foursquare-logo.png" alt="Foursquare Gospel Church Logo"></a></div>
+      <ul class="nav-menu">
+        <li class="nav-item"><a class="nav-link" href="./index.html">Home</a></li>
+        <li class="nav-item"><a class="nav-link" href="#about">About</a></li>
+        <li class="nav-item"><a class="nav-link" href="">Contact</a></li>
+        <li class="nav-item"><a class="nav-link" href="">Register</a></li>
+      </ul>
+      <div class="hamburger">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </div>
+    </nav>
+  </header>
+  <section class="about" id="about">
+    <div class="about-us">
+      <p class="theme">"Supernatural Encounter"</p>
+      <h1>FOURSQUARE ANNUAL <br> CONVENTION 2023 <br> NAMIBIA</h1>
+      <p class="about-us-description">At every convetion, we enjoy fellowship with other fellow believers from different parts of the world.
+        This is an event not to be missed. Come experience God's supernatural power. Our overseer pastor Festus 
+        Negumbo and many other Ministers of God will be ministering.
+      </p>
+      <div class="contact-us">
+        <p>Please contact us per Email for any futher questions about Foursquare Annual convetion 2023 <br>
+           <strong>foursquarechurch@gmail.com</strong>
+        </p>
+      </div>
+    </div>
+  </section>
+
+  <section id="past-projects">
+    <div class="past-projects">
+      <h4>See the past annual convetions</h4>
+      <p>Take a look at the last two Foursquare Annual convetions which took place in Namibia.</p>
+      <div class="convention-1">
```

Comment:
- [ ] I kindly suggest that you ensure the bottom is organised horizontally on desktop.


| ------------- | ------------- |
| ------------- | ------------- |
| Your Design | <img width="947" alt="image" src="https://user-images.githubusercontent.com/52934128/217905815-d3cea554-fcbc-4ec5-82c0-296951522a01.png">  |
| Template Design  | <img width="721" alt="image" src="https://user-images.githubusercontent.com/52934128/217906252-e079aaff-f550-4014-94d8-6f0ba002e801.png">  |